src/template_parser.h

```diff
@@ -138,12 +138,33 @@ struct ParserTraits<wchar_t> : public ParserTraitsBase<>
     {
         auto tmpStr = str.substr(range.startOffset, range.size());
         std::string result;
-        result.resize(tmpStr.size(), '\0');
+        const wchar_t *srcPtr;
+        size_t srcSize, destSize;
+
+        srcPtr = tmpStr.c_str();
+        srcSize = tmpStr.size();
```

Comment:
Move this declaration one line down (behind the #ifdef), please, in order to avoid gcc build warnings.